jenkins/check_file_ascii.sh

```diff
@@ -0,0 +1,34 @@
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+ERRFILE=check_ascii.sh.err
+ERRFILETEMP=$ERRFILE.0
+
+# display a helpful message for someone reading the log
+echo "Check ascii > Checking $1"
+
+if [ ! -e $1 ]; then # the file wasn't present; not necessarily an error
+    echo "WARNING: file not found: $1"
```

Comment:
Is this to account for the case of deleted files?

Reply:
Yep. I believe this logic was copies from another one of our checks.